docs/ml/upload-model.md

```diff
@@ -39,13 +39,18 @@ See [ML models in the registry](/registry/#ml-models) for more information.
 Select **Private** to only publish the model for internal use within this {{< glossary_tooltip term_id="organization" text="organization" >}}.
 Select **Next steps** to continue.
 
-{{<imgproc src="/ml/upload-model.png" resize="600x" alt="Upload model menu on the DATA tab of the Viam app.">}}
+{{<imgproc src="/ml/upload-model.png" resize="900x" alt="Upload model menu on the DATA tab of the Viam app." style="max-width: 600px">}}
 
-1. Specify a name for the model.
-2. Specify the **Model type**.
-3. Add a `.tflite` model file.
-4. (Optional) Add a `.txt` label file.
-   This should include the label names as you provided them in training, with one name per line.
+1. Select the **Model framework** or the type of model.
+   TensorFlow Lite, TensorFlow, PyTorch, and ONNX model frameworks are currently supported.
+2. Upload the files required for your model framework.
+   If you choose a TensorFlow Lite or ONNX model framework, you add a `.txt` label file.
+   This should include the label names you provided in training, with one name per line.
+
+   If you choose a TensorFlow model framework, you should upload it in the [SavedModel format](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/saved_model/README.md#the-savedmodel-format).
+
```

Comment:
I think this will be better as an unordered list within the step so they can easily see the different options. Perhaps something like this?
```suggestion
2. Upload the files required for your model framework:
   - For a TensorFlow Lite or ONNX model framework,  upload a `.txt` label file containing the label names you provided in training, with one name per line.
   - For a TensorFlow model framework, upload it in the [SavedModel format](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/saved_model/README.md#the-savedmodel-format).
```
Also [opt] consider specifying more explicitly what "it" is in the TensorFlow model framework, i.e. "...upload the model file in the [SavedModel format]..." or whatever is accurate, just so there's no ambiguity that "it" could refer to a text file as in the sentence before.

I'm suggesting "text file containing..." because that makes it clear that that's all it contains, as opposed to a text file that includes various things including the label names.